<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>706</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Scan Experiment</string>
  </property>
  <widget class="QWidget" name="central_widget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QGroupBox" name="out_group">
      <property name="title">
       <string>One Variable Sensor</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="sensor_name_label">
         <property name="text">
          <string>Sensor:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="sensor_name_box"/>
       </item>
       <item>
        <widget class="QLabel" name="delay_label">
         <property name="text">
          <string>Delay:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="delay_line"/>
       </item>
       <item>
        <widget class="QLabel" name="step_label">
         <property name="text">
          <string>Step:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="step_line"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="button_widgets" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="start_button">
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="stop_button">
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>706</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
